modules/search/instant-search/components/search-widget.jsx

```diff
@@ -14,28 +14,39 @@ import debounce from 'lodash/debounce';
  * Internal dependencies
  */
 import SearchResults from './search-results';
-import { search } from './api';
+import SearchFiltersWidget from './search-filters-widget';
+import { search, buildFilterAggregations } from '../lib/api';
 import { setSearchQuery } from '../lib/query-string';
+import { removeChildren, hideSearchHeader } from '../lib/dom';
 
 class SearchApp extends Component {
 	constructor() {
 		super( ...arguments );
+		this.input = Preact.createRef();
 		this.requestId = 0;
 		this.props.resultFormat = 'minimal';
+		this.props.aggregations = buildFilterAggregations( this.props.options.widgets );
+		this.props.widgets = this.props.options.widgets ? this.props.options.widgets : [];
```

Comment:
It's probably not a big deal since we're only doing this once in the constructor, but React Props are meant to be [read-only](https://reactjs.org/docs/components-and-props.html#props-are-read-only). 

It'd be better to use an accessor method like `SearchApp.getAggregations` or just refer to `this.props.options.widgets` when we need to access their values.